.storybook/vanillaish.js

```diff
@@ -33,7 +33,7 @@ export default create({
   inputBorderRadius: ".125rem",
 
   brandTitle: "Vanilla React Library",
-  brandUrl: "/",
+  brandUrl: "/react-components",
```

Comment:
@minkyngkm Thanks for reporting and fixing the issue. 

I think it would be more accurate if we link to Vanilla home page here. Would you mind updating this link to `https://vanillaframework.io`?

```suggestion
  brandUrl: "https://vanillaframework.io",
```